setup.py

```diff
@@ -0,0 +1,51 @@
+import os
+import subprocess
+import multiprocessing
+from distutils.core import setup
+from distutils.errors import LibError
+from distutils.command.build import build as _build
+
+class build(_build):
+    @staticmethod
+    def _configure():
+        if subprocess.call(['./configure']) != 0:
```

Comment:
If `--prefix` isn't specified, z3 grabs it from distutils, in mk_util.py (`PYTHON_PACKAGE_DIR=distutils.sysconfig.get_python_lib()`). I was torn between being more explicit about it (and passing the prefix option) vs having less to track in `setup.py`. In the end, I went with the exact procedure I used to do manual installs, but I could see it maybe being better to be explicit.